website/docs/cloud-docs/api-docs/private-registry/tests.mdx

```diff
@@ -558,7 +558,7 @@ Properties without a default value are required.
 | Key path                      | Type   | Default | Description                                                                                           |
 | ----------------------------- | ------ | ------- | ----------------------------------------------------------------------------------------------------- |
 | `data.type`                   | string | none | Must be `"vars"`.                                                                                     |
-| `data.attributes.key`         | string | none | The name of the variable.                                                                             |
+| `data.attributes.key`         | string | none | The name of the variable. Test Variable keys must begin with a letter or underscore and can only contain letters, numbers or underscores.                                                                          |
```

Comment:
Small suggestion to remove capitalization (we only capitalize product names) and making small proactive change
```suggestion
| `data.attributes.key`         | string | none | The variable's name. Test variable keys must begin with a letter or underscore and can only contain letters, numbers, and underscores.                                                                          |
```